client/signup/config/flows.js

```diff
@@ -84,6 +84,13 @@ const flows = {
 		lastModified: '2016-01-27'
 	},
 
+	subdomain: {
+		steps: [ 'design-type', 'themes', 'domains', 'plans', 'survey-user' ],
+		destination: getSiteDestination,
+		description: 'Provide a vertical for subdomains',
```

Comment:
That's correct. I don't know if there was ever a plan to expose the data, but currently it's just used as an internal reference.